docs/installation/introduction.rst

```diff
@@ -1,6 +1,125 @@
 Introduction
 ============
 
+Foreword
+^^^^^^^^
+MetalK8s is a Kubernetes_ distribution with a number of addons carefully
```

Comment:
add-ons

Reply:
"carefully picked" == chosen?
Might be removed.

Reply:
I'll rephrase with `add-ons selected for on-premises deployments, ...`